plugins/lsp/language-servers/default.nix

```diff
@@ -6,197 +6,52 @@
 }:
 with lib;
 let
-  servers = [
-    {
-      name = "ansiblels";
-      description = "ansiblels for Ansible";
-      package = "ansible-language-server";
-      cmd = cfg: [
-        "${cfg.package}/bin/ansible-language-server"
-        "--stdio"
-      ];
-    }
-    {
-      name = "ast-grep";
-      description = ''
-        ast-grep(sg) is a fast and polyglot tool for code structural search, lint, rewriting at large scale.
-        ast-grep LSP only works in projects that have `sgconfig.y[a]ml` in their root directories.
-      '';
-      serverName = "ast_grep";
-    }
-    {
-      name = "astro";
-      description = "astrols for Astro";
-      package = "astro-language-server";
-      cmd = cfg: [
-        "${cfg.package}/bin/astro-ls"
-        "--stdio"
-      ];
-    }
-    {
-      name = "basedpyright";
-      description = "basedpyright, a static type checker and language server for python";
-    }
-    {
-      name = "bashls";
-      description = "bashls for bash";
-      package = "bash-language-server";
-    }
-    {
-      name = "beancount";
-      description = "beancount-language-server";
-      package = "beancount-language-server";
-    }
-    {
-      name = "biome";
-      description = "Biome, Toolchain of the Web";
-    }
-    {
-      name = "bufls";
-      description = "Prototype for a Protobuf language server compatible with Buf.";
-      package = "buf-language-server";
-    }
-    {
-      name = "ccls";
-      description = "ccls for C/C++";
-    }
-    {
-      name = "clangd";
-      description = "clangd LSP for C/C++";
-      package = "clang-tools";
-    }
-    {
-      name = "clojure-lsp";
-      description = "clojure-lsp for Clojure";
-      serverName = "clojure_lsp";
-    }
-    {
-      name = "cmake";
-      description = "cmake language server";
-      package = "cmake-language-server";
-    }
-    {
-      name = "csharp-ls";
-      description = "csharp-ls for C#";
-      serverName = "csharp_ls";
-    }
-    {
-      name = "cssls";
-      description = "cssls for CSS";
-      package = "vscode-langservers-extracted";
-      cmd = cfg: [
-        "${cfg.package}/bin/vscode-css-language-server"
-        "--stdio"
-      ];
-    }
-    {
-      name = "dagger";
-      description = "dagger for Cuelang";
-      package = "cuelsp";
-    }
-    {
-      name = "dartls";
-      description = "dart language-server";
-      package = "dart";
+  renamedServers = {
```

Comment:
Should we split the renames into a deprecations.nix and the override / extended configs into a separate file that can be imported below?